charts/talks/README.md

```diff
@@ -0,0 +1,34 @@
+# source{d} talks deployment
+
+This is the [Helm](https://helm.sh/) chart to deploy [source{d} talks](https://talks.sourced.tech) in kubernetes.
+
+# Installing the Chart
+
+To manually install the chart, run:
+
+```bash
+$ helm install --name <release-name> <path-to-chart> --set \
+  image.tag==<label>,\
+  ingress.globalStaticIpName=<static-ip>,\
+  ingress.hosts={dns-host1}
```

Comment:
I didn't understand why it was written between curly brackets instead of <>, but I guess now it's because it's a list. I still don't understand why would we want more than one, though, but ok.